packages/fluttium/lib/src/tester.dart

```diff
@@ -169,8 +170,7 @@ class Tester {
 
   /// Wait for the semantics tree to be fully build.
   Future<void> ready() async {
-    while (_binding.pipelineOwner.semanticsOwner == null ||
-        _binding.pipelineOwner.semanticsOwner!.rootSemanticsNode == null) {
+    while (_semanticsOwner?.rootSemanticsNode == null) {
       await _binding.endOfFrame;
     }
```

Comment:
This could become an infinite while loop if semantics was never enabled, which isn't a big problem because we need it to be available but it can be prevented by just force unwrapping